### PR TITLE
python 3.13

### PR DESCRIPTION
**Pull request by lesnik512**

### CODE REVIEW

**Comment by Lancetnik** [comment]
Btw, do not forget to update pyproject as well

**Comment by Lancetnik** [comment]
Btw, what is the actual project you are using? that-depends or modern-di?

**Comment by lesnik512** [comment]
@Lancetnik for projects with sqlalchemy I'm moving to modern-di, because it's easier to manage orm sessions with scopes.

For simpler projects, where scopes are not needed, that-depends is used

**Comment by Lancetnik** [comment]
> @Lancetnik for projects with sqlalchemy I'm moving to modern-di, because it's easier to manage orm sessions with scopes.
> 
> For simpler projects, where scopes are not needed, that-depends is used

Sounds like a useless complexity, isn't it? Why are you using **that-depends** if **modern-di** is better? Do you thinking about slowly deprecation of the current lib to prior to **modern-di**?

**Comment by lesnik512** [comment]
@Lancetnik

> Sounds like a useless complexity, isn't it?

I'm panning to support both libraries, but with focus on promoting modern-di.

> Why are you using that-depends

to be fair, I don't write new projects with that-depends. I had migrated some services to test modern-di in production and it works great. I think, some day we will migrate other services to modern-di as well

File: pyproject.toml [hunk 1]
```diff
@@ -38,6 +39,18 @@ dev = [
     "typing-extensions",
     "pre-commit"
 ]
+dev313 = [
```
Comment (Lancetnik) [inline pyproject.toml:42]: I think, you can use [inlude group mechanism](https://peps.python.org/pep-0735/#dependency-group-include) here

Comment (lesnik512) [reply pyproject.toml]: found another way) with `uv sync` --no-install-package flag

**Pull request merged**